config/initializers/sphinx.rb

```diff
@@ -8,3 +8,15 @@ def indices_of_type(type)
     super.reject(&method(:inheritance_columns?))
   end
 end)
+
+# workaround unresolved issue with manticore total_pages
+# fixes SearchPresentersTest#test_pagination
+# see https://github.com/pat/thinking-sphinx/pull/1213
+ThinkingSphinx::Masks::PaginationMask.prepend(Module.new do
+  def total_pages
+    return 0 unless search.meta['total_found']
+
+    # 1000 is the default server max_matches value. We should stay at or below the server setting here.
+    @total_pages ||= ([total_entries, 1000].min / search.per_page.to_f).ceil
```

Comment:
If `1000` is the default server `max_matches` value, then `total_entries` could never be greater than `1000`, or could it?

Reply:
See blog link in description. It can show more.

Reply:
A I understand it, `total_entries` (aka `search.meta['total_found']) can be greater than `max_matches`, but `search.meta['total']` wouldn't be more than it.

But maybe I'm wrong :sweat_smile:

Reply:
Correct.